### DISCUSSION TITLE
Cookie Consent with Playwright

### DISCUSSION THREAD

**Discussion by rosellerenrqz** [orestbida/cookieconsent#711] [General]
Hello good day! So I am currently working on this cookie consent with sanity and playwright for tests, it shows up on my local machine, however, the cookie modal is not showing up upon running a playwright codegen or tests upon sanity's launch preview.

**Comment by orestbida** [comment]
@rosellerenrqz, make sure you disable the [`hideFromBots`](https://cookieconsent.orestbida.com/reference/configuration-reference.html#hidefrombots) option.
